icu4c/source/common/unicode/brkiter.h

```diff
@@ -146,7 +146,7 @@ class U_COMMON_API BreakIterator : public UObject {
      * will return distinct unequal values.
      * @stable ICU 2.0
      */
-    virtual UClassID getDynamicClassID(void) const = 0;
+    //virtual UClassID getDynamicClassID(void) const = 0;
```

Comment:
Why is this commented out? Should it be removed entirely perhaps?

Reply:
I've commented out because it is already pure virtual in the base class. I didn't know if I should remove this because the docs above. I saw getDynamicClassID commented out somewhere else in the code and thought this might be the proper way to keep the docs.
I will give a try with override = 0; thanks.
No problem, I will figure out how to handle the mistakenly landed suggestions :)